spec/system/employee_system_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Employee management', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let(:employee) { create(:employee) }
```

Comment:
instead of `create` I suggest using `build`- it will create an employee object, that is not saved in the db - so then you can use it's email in the line 17 to creation of the new account - `employee.email`

Reply:
I abandoned `let()` as it did not work in two separate examples. Instead, I used `create()` and `build()` in each of them.
Please take a look and judge if it can stay like this ;)